Jenkinsfile

```diff
@@ -21,6 +21,18 @@ node('build-slave') {
                 build_tag = sh(script: "echo " + params.github_release_tag.split('/')[-1] + "_" + commit_hash + "_" + env.BUILD_NUMBER, returnStdout: true).trim()
                 echo "build_tag: " + build_tag
 
+                stage('Customize dependencies') {
+                    if (params.WL_Customization== 'null') {
+                        println("Skipping customization")
+                    } else {
+                      sh """
+                      git clone --recurse-submodules ${WL_Customization} sunbirded-portal
+                      cp -r ${WORKSPACE}/sunbirded-portal/images/ ${WORKSPACE}/src/app/client/src/assets
```

Comment:
I think the sunbirded-portal should be SunbirdEd-portal please fix the same

Reply:
@rajeevsathish sunbirded-portal is a new folder in VM where we are cloning images and resource bundles from Adopter's private repo and then copying them from this folder to Sunbirded-portal (Cloned repo from Sunbird Ed account). So it is not necessary to keep the name of this folder exactly same.